Marble Blast Platinum/platinum/client/scripts/loadinggui.cs

```diff
@@ -375,6 +375,9 @@
 $Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: In the level editor, to make objects move on paths, add a 'Path' field to it, and put the name of the pathnode you wish for it to start on inside."; //Yoshi
 $Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: In the level editor, to make an object move on a path when the marble enters a certain area, fill the activation area with a PathTrigger, add an 'object1' field with its value set to the name of the object, and a 'Path1' field with its value set to the name of the pathnode for it to start on. To start multiple paths with the same trigger, add more fields with higher numbers, such as 'object2' and 'Path2', and so on."; //Yoshi
 $Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: In the level editor, to add water to a level, fill the space with a WaterPhysicsTrigger, then line the visible edges with WaterPlane or WaterCylinder_slow shapes."; //Yoshi
+$Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: Radar Mode 2 is useful for aligning yourself vertically with things."; //Yoshi
+$Loading::Tip[0, $Loading::Tips[0] ++] = "Tip: Radar Mode 3 is useful for getting a decent sense of your surroundings if there's a lot going on on in a large area. This is also helpful for Hunt maps with lots of gravity changes."; //Yoshi
```

Comment:
Radar modes "2" and "3" aren't very obvious what they are since the numbers are never shown in-game. This might be fine, but if we can reword each as a one to two word description of the mode, that'd be better